guides/inotify-watch-limits.md

```diff
@@ -0,0 +1,96 @@
+---
+title: Inotify Watch Limit Increase
```

Comment:
Suggest titling this "Increasing the inotify Watch Limit" or "Increasing the File Watcher Limit" (I think [inotify_add_watch](https://linux.die.net/man/2/inotify_add_watch) is typically styled in lowercase as it's the name of a system call)

```suggestion
title: Increasing the inotify Watch Limit
```

More generally, though, it seems we may want to have this be part of a larger page about tuning options (or more specifically, kernel tuning options) for Coder?